core/Network/TLS/Types.hs

```diff
@@ -19,6 +19,8 @@ module Network.TLS.Types
 import Data.ByteString (ByteString)
 import Data.Word
 
+type HostName = String
```

Comment:
I found the same code in multiple modules. I would like to remove the other definitions and let the modules import `Types`.

I guess that this redundancy is the reason why `HostName` is not highlighted in the HTML doc produced by haddock.